.circleci/config.yml

```
@@ -96,6 +110,12 @@ jobs:
             scripts/check_migrations.sh
           environment:
             TESTING_PROFILE: automated-review
+      - run:
+          name: git-secrets Scan
+          # Needs to be after a build so that the correct configuration is set
+          # by the exec-maven plugin
+          command: |
+            git secrets --scan
```

This should probably be a recursive scan - otherwise directories are not scanned

```suggestion
            git secrets -r --scan
```

---

Since we aren't providing files it uses the output of `git ls-files` to generate the list to check. You only need the `-r` if you specify a directory. You can test this by seeing the output of `git secrets --scan` and `git secrets --scan -r` are identical.

---

Sounds good - thanks for checking into that.